test/controllers/workshops_controller_test.rb

```diff
@@ -17,6 +17,24 @@ class WorkshopsControllerTest < ActionDispatch::IntegrationTest
     assert_select 'title', 'Hack Club Workshops'
   end
 
+  test 'renders root-level markdown files' do
+    get '/CONTRIBUTING.md'
+
+    # The page title should include file name and 'Hack Club Workshops'
+    assert_select 'title', 'CONTRIBUTING.md | Hack Club Workshops'
+
+    # There should be a sidebar
+    assert_select '.workshop-sidebar > ul > li > a'
+  end
+
+  test 'serves non-markdown files from root workshops directory' do
+    path = Rails.root.join('vendor', 'hackclub', 'workshops', 'test.txt')
+    FileUtils.touch(path)
+    get '/test.txt'
+    FileUtils.rm(path)
+    assert_response :success
+  end
```

Comment:
We originally  decided to not support them, but later agreed on supporting them. I need to update the PR.